helm-charts/chatqna/templates/external-llm-configmap.yaml

```diff
@@ -0,0 +1,17 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+{{- if .Values.externalLLM.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "chatqna.fullname" . }}-external-llm
+  labels:
+    {{- include "chatqna.labels" . | nindent 4 }}
+    app.kubernetes.io/component: external-llm
+data:
+  # External LLM service configuration
+  LLM_SERVER_HOST_IP: {{ .Values.externalLLM.LLM_SERVER_HOST_IP }}
+  LLM_MODEL: {{ .Values.externalLLM.LLM_MODEL }}
+  OPENAI_API_KEY: {{ .Values.externalLLM.OPENAI_API_KEY }}
+{{- end }}
```

Comment:
I think we don't need this any more since all external related configuration items are already set in the deployment.yaml